### PR TITLE
Add more info about function name

### PR DESCRIPTION
**Pull request by SOHELAHMED7**
Add more info about function name and added an example to call function locally.  I was confused where the functione-name is? So I thought to add it. Thanks.

### CODE REVIEW

**Comment by SOHELAHMED7** [comment]
`$ serverless invoke -f <function-name>`  here the function name is too generic and I was not able to find function name in 

```yml
service: app

provider:
    name: aws
    region: us-east-1
    runtime: provided

plugins:
    - ./vendor/bref/bref

functions:
    function:
        handler: index.php
        description: ''
        layers:
            - ${bref:layer.php-73}

# Exclude files from deployment
package:
    exclude:
        - 'tests/**'

```

**Comment by SOHELAHMED7** [comment]
Feel free to ask more details if more details needed for this PR

File: docs/runtimes/function.md [hunk 1]
```diff
@@ -104,6 +104,13 @@ $ serverless invoke -f <function-name>
 "Hello world"
 ```
 
+To invoke function locally
+
+```bash
+$ serverless invoke local -f <function-name>
+"Hello world"
+```
```
Comment (mnapoli) [inline docs/runtimes/function.md:112]: This part is covered by this page: https://bref.sh/docs/local-development.html

I think we can get rid of that.

File: docs/runtimes/function.md [hunk 2]
```diff
@@ -104,6 +104,13 @@ $ serverless invoke -f <function-name>
 "Hello world"
```
Comment (mnapoli) [inline docs/runtimes/function.md:104]:
```suggestion
# The function name is the one in serverless.yml, in our example that would be `hello`:
$ serverless invoke -f hello
"Hello world"
```

File: docs/runtimes/function.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ functions:
 ```
 
 The runtime (aka layer) to use is `php-XX`. To learn more check out [the runtimes documentation](/docs/runtimes/README.md).
-
+Note: In above minimal `yml` file, `hello` is the function name.
```
Comment (mnapoli) [inline docs/runtimes/function.md:91]: I suggest to move this below, I opened a suggestion inline.

**Comment by SOHELAHMED7** [comment]
I have made changes as per your suggestion. You can proceed.

**Comment by mnapoli** [comment]
Thank you for your patience!

**Pull request merged**